plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLSourceViewer.java

```diff
@@ -251,6 +255,23 @@ public void run() {
                         }
                     }, true));
             }
+            if (sourceObject.supportsObjectDefinitionOption(DBPScriptObject.OPTION_CAST_PARAMS)) {
+                toolBarManager.add(ActionUtils.makeActionContribution(
+                        new Action(SQLEditorMessages.source_viewer_cast_params_text, Action.AS_CHECK_BOX) {
```

Comment:
We probably shouldn't add two identical localizations for "Cast params" string. Let's use the same SQLEditorMessages constants everywhere.

Also I'd use "Use explicit type cast" instead.

Reply:
But you use them too:

_sql_generator_dialog_button_show_comments = Show comments
sql_generator_dialog_button_show_permissions = Show permissions

source_viewer_show_comments_text = Show comments
source_viewer_show_permissions_text = Show permissions_

How should I do correct?

Reply:
They meant you should call it differently.

Reply:
But I already changed to "Use explicit type cast"